Article Selection Policy (#1182)
w3c/wot: proposals/policies/article-selection.md

# Web of Things (WoT) Policy (Draft)

## WoT Articles Selection

The Web of Things Interest Group Marketing Task Force collects links to articles about the Web of Things on the official web page of the W3C Web of Things at [the official webpage](https://www.w3.org/WoT/about/articles/).

> The allowed articles must be from one of the following categories:

Reviewer: ```suggestion
As a default, articles should be from one of the following categories:
```

- It is written by the W3C WoT WG/IG as a whole or W3C Team. Examples are press releases
- It is written in collaboration with the WoT IG/WG where the marketing TF is the point of contact. Interest Group has reviewed it before it is published. Examples are collaborations such as the one with the JSON Schema (see [here](https://json-schema.org/blog/posts/w3c-wot-case-study)).

> - It is written by one of the member organizations or someone affiliated by one of the member organizations.

Reviewer: ```suggestion
- It is written by one of the IG or WG member organizations or someone affiliated by one of the member organizations or Invited Experts of the WoT IG or WoT WG.
```

> - It is written by one of the IG or WG member organizations or someone affiliated by one of the member organizations or Invited Experts of the WoT IG or WoT WG.

Reviewer: ```suggestion
- It is written by an official participant in the IG or WG.
```

> Thus, third-party articles are not allowed until further changes to this policy.

Reviewer: ```suggestion
Thus, third-party articles are only allowed as exceptions to the default policy and require consensus of the WoT Interest Group.
```
